scripts/institution_num_nodes.py

```diff
@@ -0,0 +1,63 @@
+import sys
+import logging
+
+from modularodm import Q
+
+from framework.transactions.context import TokuTransaction
+from scripts import utils as script_utils
+from website.app import init_app
+from website.institutions.model import Institution
+from website.project.model import Node
+
+
+logger = logging.getLogger(__name__)
+
+
+def set_institution_affiliated_nodes_count():
+    all_institutions = Institution.find()
+    for inst in all_institutions:
+        logger.info(
+            'Finding affiliated nodes for Institution with id <{}> and title <{}>'.format(inst._id, inst.name)
+        )
+        affiliated_nodes = Node.find_by_institutions(
```

Comment:
@sloria, should I be using `framework.mongo.utils.paginated()` here, or will this be okay running on production?